iphone/Maps/UI/Editor/MWMEditorViewController.mm

```diff
@@ -724,6 +724,18 @@ - (void)fillCell:(UITableViewCell * _Nonnull)cell atIndexPath:(NSIndexPath * _No
   }
 }
 
+- (void)removeAdditionalNameAtIndexPath:(NSIndexPath *)indexPath
+{
+  MWMEditorAdditionalNameTableViewCell * cell = [self.tableView cellForRowAtIndexPath:indexPath];
+  if (!cell)
```

Comment:
What is this case? Should it be asserted and debugged/fixed instead of silently ignoring?